esp/src/src-react/components/Sasha.tsx

```diff
@@ -1,96 +1,88 @@
 import * as React from "react";
-import { Dropdown, TextField, PrimaryButton } from "@fluentui/react";
+import { Dropdown, IDropdownOption, TextField, PrimaryButton } from "@fluentui/react";
 import nlsHPCC from "src/nlsHPCC";
+import { HolyGrail } from "../layouts/HolyGrail";
+import { SashaService, WsSasha } from "@hpcc-js/comms";
 
 interface SashaProps {}
 
-export const Sasha: React.FunctionComponent<SashaProps> = ({ }) => {
-  const [selectedOption, setSelectedOption] = React.useState("");
-  const [wuid, setWuid] = React.useState("");
-  const [result, setResult] = React.useState("");
+const mySashaService = new SashaService({ baseUrl: "" });
 
-  const handleOptionChange = (event: React.FormEvent<HTMLDivElement>, option: any) => {
-    setSelectedOption(option.key);
+export const Sasha: React.FunctionComponent<SashaProps> = () => {
+  const [selectedOption, setSelectedOption] = React.useState<string>("");
+  const [wuid, setWuid] = React.useState<string>("");
+  const [result, setResult] = React.useState<string>("");
+
+  const handleOptionChange = (event: React.FormEvent<HTMLDivElement>, option?: IDropdownOption) => {
```

Comment:
You could get rid of this separate function, and just have it within an onChange, like you did for the Wuid TextField. Something like: `onChange={(event, option) => setSelectedOption(option?.key as string)}` instead of the `onChange={handleOptionChange}`